UnitsNet.Tests/CustomCode/InformationTests.cs

```diff
@@ -84,5 +84,17 @@ public void OneKBHas1000Bytes()
         {
             Assert.AreEqual(1000, Information.FromKilobytes(1).Bytes);
         }
+
+        [Test]
+        public void MaxValueIsCorrectForUnitWithBaseTypeDecimal()
+        {
+            Assert.AreEqual(decimal.MaxValue, Information.MaxValue.Bits);
```

Comment:
I was so sure I went with long type on this one, getting old. I wonder why I wound up using decimal though..